requirements.txt

```
@@ -1,6 +1,7 @@
 numpy>=1.17
 scipy>=1.3
-cython>=0.29,<3
-qiskit>=0.28,<1
+cython>=3.0.5
+qiskit==1.0.0rc1
```

Remove the cap is enough, so you allow mthree to coexist with software that is not qiskit-1.0-ready
```suggestion
qiskit>=0.28
```

---

Yes, but then I cannot get the CI to chock on 1.0 things that need fixing